src/css/upcoming-events.css

```diff
@@ -0,0 +1,172 @@
+.events-header{
+  margin-bottom: 3rem;
+}
+.vertical-text{
+   writing-mode: vertical-rl;
+}
+.vertical{
+  letter-spacing: .24em;
+  font-size: 3.09rem;
+  /* padding-top: .5rem; */
+  text-align: left;
+}
+.r-line-overlay{
+  position: absolute;
+  z-index: 12;
+  top: 55;
+  left: 86%;
+}
+.card-border{
+  border-color: #eff0eb;
+}
+.events__bg-image{
+  top: -25%;
+  left: -5%;
+}
+
+
+.event-description{
+  line-height: 1.5rem;
+  width: 85%;
+}
+
+.event-date{
+  position: relative;
+  z-index: 30;
+  top: 0;
+  left: -13%;
+}
+
+.event-img{
+  width: 130px;
+  /* height: 160px; */
+}
+.event-img img{
+  width: 100%;
+}
+
+/* ============================== */
+.right-fund-card{
+  background-color: #d55343;
+}
+.r-card-font-color{
+  color: #ffffffbd;
+}
+.event-btn{
+  padding-top: 1rem !important;
+  padding-bottom: 1rem !important;
```

Comment:
Please avoid using !important at all costs, it makes the specificity very high
https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity